### PR TITLE
Add bullofheaven.com support

### LINKED ISSUE #2426: Try adding "bullofheaven.com" ?
bullofheaven.com is the site for band "Bull Of Heaven" they make Experimental/Dark Ambient/Drone tracks. not every track is playable on some media players, either the tracks are too long, or you have to do "some puzzle" to get the audio files for them.

http://bullofheaven.com/

### PR DESCRIPTION
**Pull request by inverse**
Resolves #2426

### CODE REVIEW

File: src/connectors/bullofheaven.com.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	return trackElement.textContent.replace(trackRegex, '');
+};
+
+Connector.trackArtSelector = '#jp_poster_0';
+
+Connector.currentTimeSelector = '.jp-current-time';
+
+Connector.durationSelector = '.jp-duration';
+
+Connector.isPlaying = () => !Util.isElementVisible(playButtonSelector);
```
Comment (alexesprit) [inline src/connectors/bullofheaven.com.js:24]: Why `Connector.playButtonSelector` is not used here?

File: src/connectors/bullofheaven.com.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+const playButtonSelector = '.jp-play';
+
+Connector.playerSelector = '#jp_container_N';
+
+Connector.getArtist = () => artistName;
+
+Connector.getTrack = () => {
+	const trackElement = document.querySelector('.jp-playlist-current .playlist-title');
+	return trackElement.textContent.replace(trackRegex, '');
```
Comment (alexesprit) [inline src/connectors/bullofheaven.com.js:15]: MetadataFilter is a preferred way to modify track info fields.

**Review by alexesprit — CHANGES_REQUESTED** [review]

File: src/connectors/bullofheaven.com.js (hunk 1 above)
Comment (inverse) [reply src/connectors/bullofheaven.com.js:29]: In this case it's the negative of what's expected. e.g. `playButtonSelector` is expecting it to be visible - in this case it's the reverse.

Comment (alexesprit) [reply src/connectors/bullofheaven.com.js:29]: When a track is play, the play button is expected to be hidden. The default implemetation perform the same check:
https://github.com/web-scrobbler/web-scrobbler/blob/a97bae9740438708ae85b77b8acc47940eeeaf68/src/core/content/connector.js#L333-L338

Comment (inverse) [reply src/connectors/bullofheaven.com.js]: Weird - tried this before and the logic seemed reverse. Now it's working :boom:

**Review by alexesprit — APPROVED** [review]

**Pull request merged**